README.rst

```diff
@@ -247,6 +247,53 @@ in terms of disk space.
 
 Whether ping checks are created automatically for devices.
 
+``OPENWISP_MONITORING_AUTO_DEVICE_CONFIGURATION_MODIFIED``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+-------------+
+| **type**:    | ``bool``    |
++--------------+-------------+
+| **default**: | ``True``    |
++--------------+-------------+
+
+``config_modified`` checks are run peridically to monitor whether device
+configuration is applied properly.
+This setting is used to govern whether ``config_modified`` checks are
+created automatically for newly registered devices.
+
+``OPENWISP_MONITORING_DEVICE_CONFIGURATION_MODIFIED_MAXIMUM_TIME``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+-----------+
+| **type**:    |  ``int``  |
++--------------+-----------+
+| **default**: |   ``5``   |
++--------------+-----------+
+
+This setting allows you to configure the time that the check is allowed to
+fail after which the device health status changes to ``problem``.
```

Comment:
I don't understand this setting from its explanation, we should also find a shorter name, eg `OPENWISP_MONITORING_DEVICE_CONFIG_CHECK_MAX_TIME`